docker/Dockerfile

```diff
@@ -0,0 +1,68 @@
+############################################
+# Dockerfile to run the PCIC data portal   #
+# Based on Ubuntu 16.04                    #
+############################################
+
+FROM ubuntu:16.04
+MAINTAINER Carl Masri <cmasri@uvic.ca>
+
+# Get the system level dependencies
+RUN apt-get update && apt-get install -y \
+    python-dev \
+    python-virtualenv \
+    build-essential \
+    libhdf5-dev \
+    libgdal-dev \
+    libnetcdf-dev \
+    git
```

Comment:
Yes I was noticing that as well. I'm actually going to see if I can remove the requests[security], or at least just get the minimal number of required packages to fix the pypi SSL issue.